Package/TextMate Preferences/Completions/Settings Keys.sublime-completions

```diff
@@ -77,6 +77,21 @@
 			"contents": "<key>preserveIndent</key>\n<string>$1</string>",
 			"kind": ["snippet", "s", "setting"],
 		},
+		{
+			"trigger": "indentationFoldingEnabled",
+			"contents": "<key>indentationFoldingEnabled</key>\n<false/>",
```

Comment:
As per what was told in Discord, `indentationFoldingEnabled` is `<true />` by default. Hence, I guess it doesn't make sense for the snippet to put a `<false />` as completion.

Reply:
Reason why I feel `true` makes more sense is:
- It's the default value. So having it set to `<true />` or not having the entire `<indentationFoldingEnabled><true />` is the same and hence causes the least surprise.
- The user may not know that `<true />` is the default.

Reply:
Make it `... <${1:true}/ >`, which should make (1) it clear that true is the default and (2) editing easy.